doc/source/cl/extension.rst

```diff
@@ -351,3 +352,21 @@ replaced by a customer implementation where it can be accelerated.
 
 .. _cl_khr_extended_async_copies:
   https://www.khronos.org/registry/OpenCL/specs/3.0-unified/html/OpenCL_Ext.html#cl_khr_extended_async_copies
+
+Required subgroup sizes for kernels - ``cl_intel_required_subgroup_size``
+-------------------------------------------------------------------------
+
+oneAPI Construction Kit implements the `cl_intel_required_subgroup_size`_
+extension.
+
+This has a default implementation which does not report any available subgroup
+sizes for any device. The compiler will report an error for any kernel given
+the ``intel_reqd_sub_group_size`` attribute with a size that is not reported by
+the device. There is currently no handling of the attribute if a target does
+report a set of sub-group sizes, as no in-tree target does so.
+
+Furthermore, all kernels report the same non-zero value for
+``CL_KERNEL_SPILL_MEM_SIZE_INTEL``.
```

Comment:
is this a typo? I can't work out how this fits.

Reply:
ah I see it's a strange part of the extension. why is it non-zero - the spec seems to allow for zero meaning no spill happened.

Reply:
feel free to leave as is, but I think a few extra words to say why it's a non-zero might help here.

Reply:
I think it's better for us, without any information, to assume that some spill happened, rather than baselessly guarantee to the user didn't.